ParkingZoneApp/Areas/Admin/Models/ParkingZone.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ParkingZoneApp.Areas.Admin.Models
+{
+    [Table("ParkingZone")]
+    public class ParkingZone
+    {
+        [Key]
+        [Required]
+        public Guid Id { get; set; }
+
+        [Required]
+        public string FirstName { get; set; }
+
+        [Required]
+        public string LastName { get; set; }
```

Comment:
Parking zone has FirstName and LastName? remove them just add Name

Reply:
sorry for misunderstanding